### ISSUE TITLE
Titles should be line 2 to fit bigger titles

### ISSUE THREAD

**Issue by alexookah** [tungvoduc/DTPagerController#20]
![screen shot 2017-10-20 at 11 31 01](https://user-images.githubusercontent.com/9624843/31812003-421037c0-b58a-11e7-8ede-92e67250f767.png)

How to fit bigger titles?

**Comment by tungvoduc** [comment]
@Alexookah I will check and release hotfix asap. Thanks for reporting.

**Comment by alexookah** [comment]
@tungvoduc  thanks for checking. 👍

**Comment by alexookah** [comment]
@tungvoduc any update for this?

**Comment by tungvoduc** [comment]
@Alexookah I have done something but it is not good enough to release yet. Also at the moment, I am busy with deadline at my company. I will try to update it soon.

**Comment by alexookah** [comment]
could you show what you have done... maybe we could elaborate for this.

**Comment by alexookah** [comment]
@tungvoduc could you provide me with your fix (your current progress for this)? so that i could check it by testing or try to improve/finish it ?

**Comment by tungvoduc** [comment]
@Alexookah I have not done much. What you could do is refactoring the segmentedConrol, build your own control to replace UISegmentedControl so that it would allow you to customize it into two lines or even add images. Remember to make a protocol so it is easier to replace it with a different custom control in the future.
Sorry I am very busy with our new project and currently do not have any time to contribute.

**Comment by alexookah** [comment]
i tried some way but i finally did it like this. what do you think?

```
for segment in pageSegmentedControl.subviews{
      for label in segment.subviews{
            if let labels = label as? UILabel{
                labels.numberOfLines = 2
                labels.minimumScaleFactor = 0.01
                labels.adjustsFontSizeToFitWidth = true
           }
     }
}
```

**Comment by tungvoduc** [comment]
@Alexookah I just published version 2.0.0 which supports custom segmented control. You can use your own custom segmented control from now on to support your desired UI. Please take a look at README for more detail.
